src/com/facebook/buck/apple/BuiltinApplePackage.java

```diff
@@ -99,6 +117,38 @@ public BuiltinApplePackage(
     return commands.build();
   }
 
+  private void appendAdditionalSwiftSteps(ImmutableList.Builder<Step> commands) {
```

Comment:
Yeah, I passed in the list in case we needed to add multiple commands (e.g. to support Watch). I guess in that case we could just hand back a list anyway.